activerecord/lib/active_record/connection_adapters/mysql/schema_dumper.rb

```diff
@@ -10,6 +10,10 @@ def prepare_column_options(column)
             spec[:unsigned] = "true" if column.unsigned?
             spec[:auto_increment] = "true" if column.auto_increment?
 
+            if /\A(?<size>tiny|medium|long)(?:text|blob)/ =~ column.sql_type
+              spec = { size: size.to_sym.inspect }.merge!(spec)
```

Comment:
Equivalent to `spec[:size] ||= size.to_sym.inspect` ?

Reply:
It is for pretty options ordering (`t.text :foo, size: :medium, default: "default", null: false, comment: "comment"`).